global/prometheus-alertmanager/templates/config.yaml

```diff
@@ -52,25 +52,18 @@ data:
           continue: true
           match:
             severity: critical
-      {{- range tuple "keystone" "swift" "nova" "limes" "elektra" "automation" }}
+      {{- range tuple "arc" "designate" "elektra" "keystone" "limes" "lyra" "nova" "swift" }}
       - receiver: openstack-{{ . }}
         continue: true
         match_re:
           service: {{ . }}
-          context: openstack
+          tier: openstack
       {{- end }}
       - receiver: blackbox-alerts
         continue: true
         match_re:
-          context: blackbox
-      {{- range tuple "staging" "eu-de-1" "eu-nl-1" "ap-au-1" "na-us-1" }}
-      - receiver: region_{{ . }}
-        match_re:
-          service: k8s|etcd|prometheus|dns|node
-          cluster: {{ . }}
-          severity: critical
-      {{- end }}
-
+          tier: datapath
```

Comment:
Hm. Isn't this all openstack datapath tests? I would classify this as OpenstackBlackbox*...

Maybe we should change `tier=datapath` to `tier=hardware` to make it more obvious what is meant.